public/locales/en/filters.json

```diff
@@ -7,6 +7,7 @@
   "deadlineEarliestToLatest": "Deadline (Earliest to Latest)",
   "deadlineLatestToEarliest": "Deadline (Latest to Earliest)",
   "enterMajorFilter": "Enter a major to filter by",
+  "enterSchoolFilter": "Enter a school to filter by",
   "enterStateFilter": "Enter a state to filter by",
```

Comment:
I think [interpolation ](https://www.i18next.com/translation-function/interpolation)might be a better choice but I think we would have to do something like 
```
t('key', {word: t('school')));
```

so that the phrase AND the word get translated.

Reply:
Ohh that looks nice! We should maybe do that here then